GIS.cpp

```diff
@@ -262,3 +262,20 @@ bool GIS::addEntity(std::unique_ptr<Entity> entity)
     }
     return success;
 }
+
+const Way &GIS::getWay(const EntityId &id) const {
+    Entity *entity = entities.find(id)->second.get();
+    if (entity->getType() != "Way") {
+        //TODO print to log entity is not a way
+    }
+    return *(dynamic_cast<Way*>(entity));
+}
+
+std::vector<EntityId> GIS::getWaysByJunction(const EntityId &id) const {
+    Entity *entity = entities.find(id)->second.get();
```

Comment:
"The function shall get EntityId of a Junction and return a vector of Ids of all the ways that start at this junction and those which end at this junction and are bidirectional"

Reply:
fixed